README.md

```diff
@@ -1,4 +1,7 @@
+This is a fork of [biblatex-csl-converter](https://www.npmjs.com/package/biblatex-csl-converter) which adds TypeScript and crossref support. The below README is taken from the original package.
+
```

Comment:
@DerDrodt We don't need this paragraph any more, do we?

Reply:
Yeah, good catch. Fixed now